srcpkgs/font-Siji/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'font-Siji'
+pkgname=font-Siji
+version=0.0.180725
```

Comment:
git packages use 0.0.YYYYMMDD

Reply:
https://github.com/voidlinux/void-packages/pull/6707#discussion_r121602775
https://github.com/voidlinux/void-packages/pull/6707#discussion_r121899848

`0.0.0.YYYYMMDD` is probably better.